samples/ResourceOwnerPasswordFlow/.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig is awesome:
```

Comment:
Well, that's a comment that comes with the editor.config file. I have it there for use with the C# codeformatter. https://github.com/dotnet/codeformatter